sct.py

```diff
@@ -210,8 +214,13 @@ def provision_resources(backend, test_name: str, config: str):
         click.echo("No need provision logging service")
 
     click.echo(f"Provision {backend} cloud resources")
-    layout = SCTProvisionLayout(params=params)
-    layout.provision()
+    if backend == "aws":
+        layout = SCTProvisionLayout(params=params)
+        layout.provision()
```

Comment:
just a question, would this be in the same format as in `backend == "azure"` in the future?

Reply:
yes, in future all provisioning will be unified.